tests/api/test_general.py

```diff
@@ -130,7 +122,7 @@ def test_get_on_model_wo_uid(client, resource):
     rv = client.get('/api/v1/{0}/some_repo'.format(resource))
     assert rv.status_code == 400
     invalid_msg = ('The requested resource "{0}" is invalid. Choose from the following: '
-                   'advisory, advisorystate, bugzillabug, containeradvisory, containerkojibuild, '
+                   'advisory, bugzillabug, containeradvisory, containerkojibuild, '
                    'distgitcommit, freshmakerevent, kojibuild, kojitag, kojitask, and user.'
```

Comment:
Optional: You could move some of the start of this string to the end of the line above